versions.mk

```diff
@@ -0,0 +1,38 @@
+BUILDX_VERSION := 0.11.2
+CERT_MANAGER_VERSION := v1.13.1
+CHART_TESTING_VERSION := 3.9.0
+CONTAINER_STRUCTURE_TEST_VERSION := 1.16.0
+HELM_DOCS_VERSION := 1.11.2
+HELM_VERSION := 3.13.0
+# kind node image version is related to kind version.
+# if you change kind version, also change kind node image version.
+KIND_VERSION := v0.20.0
+# It is set by CI using the environment variable, use conditional assignment.
+KUBERNETES_VERSION ?= 1.27.3
+MINIKUBE_VERSION := v1.31.2
+PROTOC_VERSION :=  24.4
+
+# Tools versions which are defined in go.mod
+CONTROLLER_RUNTIME_VERSION := $(shell awk '/sigs\.k8s\.io\/controller-runtime/ {print substr($$2, 2)}' go.mod)
+CONTROLLER_TOOLS_VERSION := $(shell awk '/sigs\.k8s\.io\/controller-tools/ {print substr($$2, 2)}' go.mod)
+GINKGO_VERSION := $(shell awk '/github.com\/onsi\/ginkgo\/v2/ {print $$2}' ../go.mod)
```

Comment:
```suggestion
GINKGO_VERSION := $(shell awk '/github.com\/onsi\/ginkgo\/v2/ {print $$2}' go.mod)
```

Reply:
Since the path to `go.mod` is specified by a relative path, whether this variable can be used as intended in the Makefile that includes `version.mk` depends on where the Makefile is located.

Reply:
I have found that both `go.mod` and `../go.mod` have problems:
- If we use `go.mod`, then `make -C e2e COMMAND` fails (with the following error message: `awk: fatal: cannot open file 'go.mod' for reading: No such file or directory`)
- If we use `../go.mod`, then `make COMMAND` fails (with the following error message: `awk: fatal: cannot open file '../go.mod' for reading: No such file or directory`)

Reply:
I consider the problem can be solved by this trick.
https://github.com/topolvm/topolvm/pull/781/files#diff-995146baf6475898321c6ef6dbdeef8b1dc91c6f4a6cdff87dc13645617feb06R3
I'll dig it more.